### PR TITLE
Add statsd timer collection for http, get reputation, and redis commands

### PR DESCRIPTION
**Pull request by ajvb**
Adds optional statsd metric collection, with a generic http timer, a get reputation timer, and a timer for all redis commands.

### CODE REVIEW

File: iprepd.go [hunk 1]
```diff
@@ -41,6 +42,9 @@ type serverCfg struct {
 		AWS  bool
 	}
 	VersionResponse string
+	Statsd          struct {
```
Comment (ameihm0912) [inline iprepd.go]: Looks like this gofmt here is off?

File: redis.go [hunk 2]
```diff
@@ -55,6 +65,7 @@ func newRedisLink(cfg serverCfg) (ret redisLink, err error) {
 		WriteTimeout: time.Millisecond * time.Duration(cfg.Redis.WriteTimeout),
 		DialTimeout:  time.Millisecond * time.Duration(cfg.Redis.DialTimeout),
 	})
+	ret.master.WrapProcess(instrumentRedisCmd)
```
Comment (ameihm0912) [inline redis.go]: Is the performance impact of wrapping every redis command with a statsd call fairly negligible?

File: iprepd.go (hunk 1 above)
Comment (ajvb) [reply iprepd.go]: No, but it does look weird. It's just being aligned with the `VersionResponse` above it.

File: redis.go (hunk 2 above)
Comment (ajvb) [reply redis.go]: Yes

**Pull request merged**